components/server/src/installation-admin/installation-admin-controller.ts

```diff
@@ -50,10 +50,15 @@ export class InstallationAdminController {
 
             log.info(`${adminUserCreateLoginTokenRoute} received.`);
             try {
+                // In case there is no/an empty key specified: Nobody should be able to call this so they are not able to guess values here
+                if (!this.config.admin.loginKey) {
+                    throw new Error("Cannot handle request");
+                }
+
                 // Unblock the admin-user: it's blocked initially!
                 const user = await this.userDb.findUserById(BUILTIN_INSTLLATION_ADMIN_USER_ID);
                 if (!user) {
-                    throw new Error("Cannot find builint admin-user");
+                    throw new Error("Cannot find builtin admin-user");
```

Comment:
Use  `BUILTIN_INSTLLATION_ADMIN_USER_ID` to reference the user const

Reply:
I feel the ID is too technical, and referenced the user by it's name throughout user-facing strings. :+1: